pkg/minikube/bootstrapper/kubeadm/kubeadm.go

```diff
@@ -119,12 +119,17 @@ func (k *KubeadmBootstrapper) StartCluster(k8s config.KubernetesConfig) error {
 		KubeadmConfigFile   string
 		SkipPreflightChecks bool
 		Preflights          []string
+		DNSAddon            string
```

Comment:
Can you elaborate in the PR description or commit message on why this change is necessary?

kube-Perhaps something about the default changing from kube-dns to coredns in 1.12?

Reply:
Updated PR description. I had a link to comment #3164 (comment) that explains the reasons, but agree to have it in PR is better.